system/View/Parser.php

```diff
@@ -425,6 +430,58 @@ protected function parseComments(string $template): string
 	{
 		return preg_replace('/\{#.*?#\}/s', '', $template);
 	}
+	
+	/**
+	 * Replaces html colors with temporary value so it does not affect parsing regex
+	 * 
+	 * Takes ..color:#fff, background-color:#993399 and color=#0284D0> 
+	 * Returns ..color{{color:fff}}, background-color{{color:993399}}, and color{{color=0284D0}}
+	 * 
+	 * @param string $template
+	 * 
+	 * @return string
+	 */
+	function parseHtmlColors(string $template) : string
```

Comment:
please add modifier, you can use private for it: 

```suggestion
	private function parseHtmlColors(string $template): string
```